.github/workflows/rust-audit.yml

```diff
@@ -0,0 +1,28 @@
+name: Rust security audit
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+  push:
+    paths:
+      - '**/Cargo.toml'
+      - 'Cargo.lock'
+
+jobs:
+  security-audit:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/audit-check@v1.2.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'd prefer to use shell commands or actions-rs/cargo instead of audit-check because there is this limitation: https://github.com/actions-rs/clippy-check/issues/2 (It is PR that actually adds the dependency)

Reply:
Then we dont get automatic issue generation. Each way of doing it would have its own limitations. @taiki-e, have you considered this?

Reply:
I wonder if it might make sense to have this in two separate workflow files? Using the `audit-check` action in the daily check will always be running in the context of the tokio-rs org and so it will always have the token available. And then not using `audit-check` in jobs in response to a changed `Cargo.toml`  because that _will_ be run in pull requests where that issue will then be problematic.

Reply:
> Then we dont get automatic issue generation. Each way of doing it would have its own limitations. @taiki-e, have you considered this?

Oh, I was missing it. Thanks for pointing that out!

> Using the `audit-check` action in the daily check will always be running in the context of the tokio-rs org and so it will always have the token available. And then not using `audit-check` in jobs in response to a changed `Cargo.toml` because that _will_ be run in pull requests where that issue will then be problematic.

This seems to make sense to me 👍 
(Probably, actually, it doesn't necessarily have to be two workflows, we can also set two mutually exclusive steps. I'm fine with either way.)